django-backend/fecfiler/web_services/dot_fec/dot_fec_composer.py

```diff
@@ -80,7 +80,7 @@ def __init__(
         self.soft_name = soft_name
         self.soft_ver = soft_ver
         self.rpt_id = rpt_id
-        self.rpt_number = (rpt_number,)
+        self.rpt_number = rpt_number
```

Comment:
very interesting!  i wonder how that happened

Reply:
same.  my first thought was eslint got it somehow?